### PR TITLE
Spring Rest Docs Research for our Runtime Bundles

### PR DESCRIPTION
**Pull request by lucianoprea**
- Added spring-rest-docs to cloud-services-rest to document the API (implemented only for HomeController as a PoC)
- updating poms for lucianoprea_1497_spring_rest_docs version

This is a PoC for [#1497](https://github.com/Activiti/Activiti/issues/1497)

PR [#4](https://github.com/Activiti/activiti-cloud-build/pull/4) opened in cloud-services-build is part of this work

### CODE REVIEW

**Comment by Salaboy** [comment]
@lucianoprea this is great stuff.. Please add tests for process definitions and instances.

**Comment by erdemedeiros** [comment]
Please, take a look on the Codacy report as well. I disabled the check about assert on tests, as it does not make sense here.

File: activiti-cloud-services-runtime-bundle/activiti-cloud-services-rest-impl/pom.xml [hunk 1]
```diff
@@ -81,7 +85,46 @@
...
+      <artifactId>spring-restdocs-mockmvc</artifactId>
+      <scope>test</scope>
+    </dependency>
   </dependencies>
 
+  <build>
+    <plugins>
+      <plugin>
+        <groupId>org.asciidoctor</groupId>
+        <artifactId>asciidoctor-maven-plugin</artifactId>
```
[the remark this answers is not in the document]
Comment (lucianoprea) [reply activiti-cloud-services-runtime-bundle/activiti-cloud-services-rest-impl/pom.xml]: Fix that, moved to the _activiti-cloud-build_ and added the version there

File: activiti-cloud-services-runtime-bundle/activiti-cloud-services-rest-impl/src/main/asciidoc/index.adoc [hunk 2]
```diff
@@ -0,0 +1,198 @@
...
+
+== Home 
+
+.Request
+include::{snippets}/home/http-request.adoc[]
+
+.Response
+include::{snippets}/home/http-response.adoc[]
+
+.Links
```
[the remark this answers is not in the document]
Comment (lucianoprea) [reply activiti-cloud-services-runtime-bundle/activiti-cloud-services-rest-impl/src/main/asciidoc/index.adoc]: This is here as a PoC

**Pull request merged**